pkgs/applications/editors/sublime/4/common.nix

```diff
@@ -0,0 +1,153 @@
+{ buildVersion, x64sha256, dev ? false }:
+
+{ fetchurl, stdenv, xorg, glib, libglvnd, glibcLocales, gtk3, cairo, pango, libredirect, makeWrapper, wrapGAppsHook
+, pkexecPath ? "/run/wrappers/bin/pkexec"
+, writeScript, common-updater-scripts, curl, jq, gnugrep
+, openssl, bzip2, bash, unzip, zip
+}:
+
+let
+  pname = "sublimetext4";
+  packageAttribute = "sublime4${stdenv.lib.optionalString dev "-dev"}";
+  binaries = [ "sublime_text" "plugin_host-3.3" "plugin_host-3.8" "crash_reporter" ];
+  primaryBinary = "sublime_text";
+  primaryBinaryAliases = [ "subl" "sublime" "sublime4" ];
+  downloadUrl = "https://download.sublimetext.com/sublime_text_build_${buildVersion}_${arch}.tar.xz";
+  versionUrl = "https://www.sublimetext.com/updates/4/dev_update_check";
+  versionFile = builtins.toString ./packages.nix;
```

Comment:
I'd name this `versions` since it's not a file

Reply:
It is a path to a file.

Reply:
Oh, it expands the path? Now I get it, sorry for the noise

Reply:
Yeah, it is used by the update script.